pyaedt/generic/farfield_visualization.py

```
@@ -0,0 +1,2187 @@
+                }
+
+                # Iterate over Power elements
+                power_info = source.find("PowerInfo")
+                if power_info is not None:
+                    source_info["incident_power"] = {}
+                    source_info["accepted_power"] = {}
+                    source_info["radiated_power"] = {}
+                    for power in power_info.findall("Power"):
+                        freq = power.get("Freq")
+                        source_info["incident_power"][freq] = {}
+                        source_info["incident_power"][freq] = power.find("IncidentPower").text.strip()
+                        source_info["accepted_power"][freq] = {}
+                        source_info["accepted_power"][freq] = power.find("AcceptedPower").text.strip()
+                        source_info["radiated_power"][freq] = {}
+                        source_info["radiated_power"][freq] = power.find("RadiatedPower").text.strip()
+
+                sources.append(source_info)
+                cont += 1
+        return sources
+
+    @pyaedt_function_handler()
+    def __create_geometries(self, export_path):
+        """Export the geometry in OBJ format."""
+        self.__app.logger.info("Exporting geometry...")
+        model_pv = self.__app.post.get_model_plotter_geometries(plot_air_objects=False)
+        obj_list = {}
+        for obj in model_pv.objects:
+            object_name = os.path.basename(obj.path)
+            name = os.path.splitext(object_name)[0]
+            original_path = os.path.dirname(obj.path)
+            new_path = os.path.join(os.path.abspath(export_path), object_name)
+
+            if not os.path.exists(new_path):
+                new_path = shutil.move(obj.path, export_path)
+            if os.path.exists(os.path.join(original_path, name + ".mtl")):  # pragma: no cover
+                try:
+                    os.remove(os.path.join(original_path, name + ".mtl"))
```

Try to use (shutil.rmtree)[https://docs.python.org/3/library/shutil.html#shutil.rmtree] instead, you can ignore errors with an argument